packages/mui-material/src/SwipeableDrawer/SwipeableDrawer.d.ts

```diff
@@ -1,7 +1,26 @@
 import * as React from 'react';
-import { DrawerProps } from '../Drawer';
+import { DrawerProps, DrawerSlotsAndSlotProps } from '../Drawer';
 
-export interface SwipeableDrawerProps extends Omit<DrawerProps, 'onClose' | 'open'> {
+type SwipeableDrawerSlotsAndSlotProps = DrawerSlotsAndSlotProps & {
```

Comment:
`export interface SwipeableDrawerSwiperAreaSlotPropsOverrides {}`

I think this is missing

Reply:
Added